### PR TITLE
Create Iteration2 Freedom Toy logo

### LINKED ISSUE #70: Make changes regardinf review of logos - Iteration 2
- [ ] make changes
- [ ] merge PR

______________

This issue size:
- Estimate SP: 3
- Actual SP: ?

### PR DESCRIPTION
**Pull request by naila135**
Resolve #70

### CODE REVIEW

File: PublisherBrand/Iterations/Iteration2.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+This is second iteration of creating logo for Freedom Toys.<br>
```
Comment (sahlet-official) [inline PublisherBrand/Iterations/Iteration2.md:1]: Change to FreedomToy please

**Review by sahlet-official — CHANGES_REQUESTED** [review]

File: PublisherBrand/Iterations/Iteration2.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+After [Iteration1](./Iteration1.md) review was made.<br>
+Several changes were conducted:
+- Product Owner wished to make dark variations for all logos from iteration1;
+- Product Owner asked designer to make some lines wider;
+- Product Owner asked to implement a new sketch;
+
+This iteration results can be seen [here](https://www.figma.com/file/ciA7yOiSqzYXabkYPZtjjy/FreedomToys?node-id=32-2)
```
Comment (sahlet-official) [inline PublisherBrand/Iterations/Iteration2.md:8]: put a dot after the sentence

**Review by sahlet-official — CHANGES_REQUESTED** [review]

File: PublisherBrand/Iterations/Iteration2.md (hunk 1 above)
Comment (naila135) [reply PublisherBrand/Iterations/Iteration2.md]: done

File: PublisherBrand/Iterations/Iteration2.md (hunk 2 above)
Comment (naila135) [reply PublisherBrand/Iterations/Iteration2.md]: done

**Review by sahlet-official — APPROVED** [review]

**Pull request merged**